nnfabrik/utility/nn_helpers.py

```diff
@@ -129,6 +129,7 @@ def load_state_dict(
     match_names: bool = False,
     ignore_dim_mismatch: bool = False,
     prefix_agreement: float = 0.98,
+    strict=None,
```

Comment:
```suggestion
    strict=True,
```

To keep default behavior